docs/griptape-framework/drivers/web-search-drivers.md

```diff
@@ -21,12 +61,6 @@ Example using `GoogleWebSearchDriver` directly:
 --8<-- "docs/griptape-framework/drivers/src/web_search_drivers_1.py"
 ```
 
-Example of using `GoogleWebSearchDriver` with an agent:
-
-```python
---8<-- "docs/griptape-framework/drivers/src/web_search_drivers_2.py"
-```
-
```

Comment:
what happened to this example?

Reply:
I had instructed @william-price01 to replace "default" example with a DDG one.